src/Symfony/Bridge/Doctrine/Logger/DbalLogger.php

```diff
@@ -28,6 +30,8 @@ class DbalLogger implements SQLLogger
 
     public function __construct(LoggerInterface $logger = null, Stopwatch $stopwatch = null)
     {
+        trigger_deprecation('symfony/doctrine-bridge', '6.4', '"%s" is deprecated, use a middleware instead.', __CLASS__);
```

Comment:
would it make sense to trigger this before the class declaration?

Reply:
Yeah, probably.

Reply:
Done.